src/App.js

```diff
@@ -1,25 +1,93 @@
 import { useEffect } from 'react'
-import logo from './logo.svg'
+import { Link, Route, Switch } from 'react-router-dom'
+import About from './component/page/about/About'
+import Home from './component/page/home/Home'
+import tiktokIcon from "./assets/tiktok.com.png"
+import facebookIcon from "./assets/facebook.com.png"
+import instagramIcon from "./assets/instagram.com.png"
+import Author from './component/authorImage/Author'
+
 
 function App() {
 
   useEffect(() => {
     const circleInner = document.querySelector(".circle-inner")
     const circleOuter = document.querySelector(".circle-outer")
 
+    const liNav = document.querySelectorAll(".nav-li")
+    const cursorNav = document.querySelector(".cursor")
+
     document.addEventListener("mousemove", (e) => {
       circleInner.style.top = e.pageY + "px"
       circleInner.style.left = e.pageX + "px"
 
       circleOuter.style.top = e.pageY + "px"
       circleOuter.style.left = e.pageX + "px"
     })
+
+    liNav.forEach((li, index) => {
+      li.addEventListener("mouseover", () => {
+        cursorNav.style.left = index * 100 + "px"
+      })
+    })
+
```

Comment:
I'm not sure if it is appropriate now because for now, this is just a frame, we don't put any effect change by user, so for me, let's remove the animation here first, then when you working on this part, it will more clearly and make reviewer understood about what you're working on